modules/tinymce/tools/docs/tinymce.editor.ui.Ui.js

```diff
@@ -35,6 +35,24 @@
  * @method tinymce.editor.ui.hide
  */
 
+/**
+ * Enables the editor user interface. This method affects all user interface
+ * elements, including: menu bar, toolbar, notifications, and dialogs. Can
+ * not be used when in readonly mode.
+ * <br>
+ * <em>Added in TinyMCE 5.6</em>
+ *
+ * @method tinymce.editor.ui.enable
+ */
+
+/**
+ * Disables the editor user interface. This method affects all user interface
+ * elements, including: menu bar, toolbar, notifications, and dialogs.
+ * <em>Added in TinyMCE 5.6</em>
```

Comment:
Should there be a `<br>` before this also?

Reply:
Yeah, this needs a `<br>` to break up the content.